### PR TITLE
Add page on diversity and inclusion

### PR DESCRIPTION
**Pull request by joshuaulrich**
Add page similar to what's on cvent.

### CODE REVIEW

File: content/diversity.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Other than keynotes, all talks are given by people who respond to the Call for
+Presentations. To increase diversity of the presentations and presenters at the
+R/Finance conference, the committee asks and encourages members of under-
+represented or historically marginalized groups to respond to the Call for
+Presentations. Even if you don't consider yourself a member of one of these
+groups, you can help by encouraging your peers and colleagues in these groups
+to respond to the Call for Presentations.
+
+Speaking at a conference can be intimidating, especially if it's your first
```
Comment (eddelbuettel) [inline content/diversity.md]: Small nit:  I prefer "it is" in written English.

File: content/diversity.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Diversity and Inclusion"
+draft: false
+---
+
+The lack of diversity in tech is well-known, and the situation in quantitative
+finance is potentially even worse than in tech. There has been great work on
```
Comment (eddelbuettel) [inline content/diversity.md]: s/work/progress/ ?

File: content/diversity.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+represented or historically marginalized groups to respond to the Call for
+Presentations. Even if you don't consider yourself a member of one of these
+groups, you can help by encouraging your peers and colleagues in these groups
+to respond to the Call for Presentations.
+
+Speaking at a conference can be intimidating, especially if it's your first
+time. You can request your talk be a 5-7 minute "lightning talk" if a full
+talk would be uncomfortable. Also, a full paper is not required. We love talks
+from practitioners who share their experience solving problems on-the-job. You
+don't have to be well-known, or an expert, to provide a valuable contribution.
```
Comment (eddelbuettel) [inline content/diversity.md]: Same: "do not"

File: content/diversity.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Travel/accommodation funding is available if that would prevent you from
```
Comment (eddelbuettel) [inline content/diversity.md]: "can be made available upon request"

**Comment by joshuaulrich** [comment]
Thanks for the feedback Dirk. I made your changes, with one difference (s/made available/provided/).

**Pull request merged**